internal/controller/deployment/integrations/kubernetes/namespace_handler.go

```diff
@@ -46,7 +48,7 @@ func NewNamespaceHandler(kubernetesClient client.Client) dataplane.ResourceHandl
 }
 
 func (h *namespaceHandler) Name() string {
-	return "KubernetesNamespace"
+	return NamespaceHandlerName
```

Comment:
We can keep the string here without a const as it is only used in one place.

Reply:
reverted this change